packages/rest/src/router/external-express-routes.ts

```diff
@@ -90,6 +126,24 @@ class ExternalRoute implements RouteEntry, ResolvedRoute {
   }
 }
 
+export function rebaseOpenApiSpec<T extends Partial<OpenApiSpec>>(
```

Comment:
Please add few unit-level tests to verify how `rebaseOpenApiSpec` handles different edge cases. Take a look at `if` conditions below to see what to test.